compliance/flush.go

```diff
@@ -179,12 +183,13 @@ func FlushOfAllNIs(c *fluent.GRIBIClient, wantACK fluent.ProgrammingResult, t te
 
 	checkNIHasNEntries(ctx, c, defaultNetworkInstanceName, 0, t)
 	checkNIHasNEntries(ctx, c, vrfName, 0, t)
+	flushServer(c, t)
 }
 
 // FlushServer flushes all the state on the server, but does not validate it
-// specifically. It can be called from external tests that need to clean up
+// specifically. It can be called from tests that need to clean up
 // a server between test cases.
-func FlushServer(c *fluent.GRIBIClient, t testing.TB) {
```

Comment:
I'm in favor of letting the test driver do the flushing rather than doing it in individual tests, so we know it's done consistently before each test, not after. Since if a test fails, it would not get the chance to flush.

Reply:
I think that we can overcome this issue by specifying `defer flushServer(c, t)` at the start of each of the tests. I think a deferred call will be run even with `t.Fatalf`.